api/serializers.py

```diff
@@ -135,6 +136,21 @@ def create(self, validated_data):
             for disease in validated_data['diseases']:
                 classifier.diseases.add(disease)
 
+        if settings.CREATE_TASKS:
+            task_service = TaskServiceClient(settings.TASK_SERVICE_BASE_URL,
+                                             settings.AUTH_TOKEN)
+
+            task = {
+                'task_def': 'classifier-search',
```

Comment:
This line is causing tests to fail when running via docker-compose. The problem is that this specifies a task definition by name, but if that task definition doesn't exist then the request 400s. Should the task-service automatically create a new task-def if one doesn't exist, or should core-service be responsible for pre-creating  task-def to attach to the task?